### PR TITLE
FUSETOOLS-2565 update to FIS 2.0 R3

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java [hunk 1]
```diff
@@ -58,16 +58,17 @@
 	
 	public static final String FIS_20_R0_CAMEL_VERSION = "2.18.1.redhat-000012";
 	public static final String FIS_20_R1_CAMEL_VERSION = "2.18.1.redhat-000015";
+	public static final String FIS_20_R3_CAMEL_VERSION = "2.18.1.redhat-000021";
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java]: wondering what happened to FIS_20_R2_CAMEL_VERSION

**Comment by lhein** [comment]
Is that version already on a public repo?

**Comment by apupier** [comment]
> Is that version already on a public repo?

yes

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java (hunk 1 above)
Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java]: They jumped a version due to some issues but can't find a note where it is written

Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java]: a not very clear document but which is the best: https://access.redhat.com/documentation/en-us/red_hat_jboss_fuse/6.3/html-single/fuse_integration_services_2.0_for_openshift/#patch-apps-bom
Jboss Fuse Rollup 4 is FIS 2.0 R3

Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java]: ok I understood: the numbering was wrong, we have never provided  something for FIS 2.0 R0, we started with FIS 2.0R1.

**Pull request merged**